clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java

```diff
@@ -496,8 +502,38 @@ public Map reconcile(String cloudProvider, String account, String region, boolea
       results.put("deletedFromElasticsearch", true);
 
       if (deleteFromSource) {
-        Lists.partition(entityTagsForNamespace, 50).forEach(front50Service::batchUpdate);
-        results.put("deletedFromSource", true);
+        ExecutorService executor =
+            Executors.newFixedThreadPool(
+                6,
```

Comment:
should this be a configurable option?

Reply:
i was thinking about it, but since NFLX is pretty much the only company using entity tags and it's an "admin" operation I figured it wasn't worth the added config, what do you think?

Reply:
fair enough